test/test_manifest.json

```diff
@@ -7530,5 +7530,17 @@
       "md5": "1aa34fbb2154f9a647c7fa9e90db0eff",
       "rounds": 1,
       "type": "text"
+   },
+   {
+      "id": "annotation-fileattachment-forced-colors-eq",
+      "file": "pdfs/annotation-fileattachment.pdf",
+      "md5": "9a192d8b1a7dc652a19835f6f08098bd",
```

Comment:
This fails testing locally, since that's the hash of the `tracemonkey.pdf` file. Surely this is what you meant:
https://github.com/mozilla/pdf.js/blob/797f8d3dca5e0770c83788b3b65387884031caa0/test/test_manifest.json#L6160